packages/data-service/src/connect-mongo-client.spec.ts

```diff
@@ -143,7 +155,7 @@ describe('connectMongoClient', function () {
 
     it('should run the ping command with the specified ReadPreference', async function () {
       const commands: CommandStartedEvent[] = [];
-      const [metadataClient, crudClient] = await connectMongoClient(
+      const [metadataClient, crudClient, , state] = await connectMongoClient(
```

Comment:
Does the extra comma mean we ignore the `tunnel` item? I don't think I've seen this syntax used before, cool.

Reply:
Yep, it just skips the item :+1: